service/storage_proxy.cc

```diff
@@ -4995,14 +5006,15 @@ storage_proxy::handle_write(netw::messaging_service::msg_addr src_addr, rpc::opt
                         f.ignore_ready_future();
                     });
                 }).handle_exception([reply_to, shard, &p, &errors] (std::exception_ptr eptr) {
+                    errors.count++;
+                    errors.local = replica::try_encode_replica_exception(eptr);
                     seastar::log_level l = seastar::log_level::warn;
-                    if (is_timeout_exception(eptr)) {
-                        // ignore timeouts so that logs are not flooded.
+                    if (is_timeout_exception(eptr) || std::holds_alternative<replica::rate_limit_exception>(errors.local.reason)) {
+                        // ignore timeouts and rate limit exceptions so that logs are not flooded.
```

Comment:
Here as well, the comment below claims that for timeouts a counter was incremented. What about rate limit?

Reply:
There is now an appropriate counter for rate limit exceptions and it is incremented. Updated the comment so that it mentions it.